### PR TITLE
getting_started.org: Gentoo Linux install guide

### PR DESCRIPTION
**Pull request by mjkalyan**
Added installation instructions including Emacs & dependencies for Gentoo/Portage users to the getting_started.org file for the sake of completeness.

### CODE REVIEW

**Comment by TheGreatMcPain** [comment]
You might want to mention that the `xft` use flag must be enabled for `app-editors/emacs` in order for the fonts to render correctly.

See #4876

File: docs/getting_started.org [hunk 1]
```diff
@@ -224,6 +225,46 @@ zypper install rust
...
+To use Emacs graphically, enable the =gui= USE flag. And enable the =xft= USE flag to render fonts correctly (see
+[[https://github.com/hlissner/doom-emacs/issues/4876][issue #4876]])
+#+begin_src sh
+echo "app-editors/emacs gui xft" >> /etc/portage/package.use/emacs
+#+end_src
+
+To install the latest unmasked version compatible with Doom:
+#+begin_src sh
+emerge '>=app-editors/emacs-26.3'
+#+end_src
```
Comment (hlissner) [inline docs/getting_started.org:241]: Can this be updated with 27.x installation instructions? Doom no longer officially supports 26.x.

**Review by hlissner — CHANGES_REQUESTED** [review]

File: docs/getting_started.org (hunk 1 above)
Comment (mjkalyan) [reply docs/getting_started.org]: Done!

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Looks good to me. Thanks!

**Pull request merged**